fabric/src/main/java/io/github/foundationgames/automobility/fabric/controller/controlify/ControlifyCompat.java

```diff
@@ -0,0 +1,71 @@
+package io.github.foundationgames.automobility.fabric.controller.controlify;
+
+import dev.isxander.controlify.api.ControlifyApi;
+import dev.isxander.controlify.api.bind.BindingSupplier;
+import dev.isxander.controlify.api.bind.ControllerBinding;
+import dev.isxander.controlify.rumble.BasicRumbleEffect;
+import dev.isxander.controlify.rumble.ContinuousRumbleEffect;
+import dev.isxander.controlify.rumble.RumbleSource;
+import dev.isxander.controlify.rumble.RumbleState;
+import io.github.foundationgames.automobility.controller.ControllerCompat;
+
+public class ControlifyCompat implements ControllerCompat {
+    static BindingSupplier accelerateBinding, brakeBinding, driftBinding;
```

Comment:
Is there a reason for package-private here?

Reply:
It's just the strictest visibility used. If you'd prefer public?

Reply:
If they are used outside of that class in that package, then it can be left as-is, otherwise private would be fine